packages/cli-kit/src/private/node/conf-store.ts

```diff
@@ -3,12 +3,19 @@ import Conf, {Schema} from 'conf'
 
 export {Conf, Schema}
 
+interface Cache {
+  [key: string]: {
+    value: unknown
+    timestamp: number
+  }
+}
+
 export interface ConfSchema {
   sessionStore: string
+  cache: Cache
```

Comment:
I'm a bit concerned with the approach we are taking here. Cache's schema is very loose, which makes it the perfect place to store any value with a timestamp. If that ends up happening, it won't be straightforward to know what state is stored in it, and thus, which changes that we are shipping are not breaking whatever state exists there.

I recommend keeping the configuration schema flat and strong so instead of calling this attribute `cache`, I'd call it `identityIntrospectionEndpointCachedValue` and **make it optional**. Introducing schema values that are non-optional is a breaking change.